src/index.ts

```diff
@@ -333,7 +331,11 @@ export function startAnalysis<
             });
         },
         startAppAnalysis: async (appPath, options) => {
-            const appPathMain = getAppPathMain(appPath);
+            if (typeof appPath !== 'string' && analysisOptions.platform !== 'android')
+                throw Error('Could not install app: Split app files are only supported on Android.');
+
+            // This might not be the main APK, but we don‘t care because we should get the same meta information out of all the APKs
```

Comment:
For the app ID and version code, that is true. For everything else, it isn't, unfortunately.

See: https://github.com/tweaselORG/appstraction/issues/4#issuecomment-1485007859